app/views/support_interface/candidates/show.html.erb

```diff
@@ -8,6 +8,19 @@
   }) %>
 <% end %>
 
+<% if FeatureFlag.active?(:duplicate_matching) %>
+  <div class="govuk-body">
+    <% if @candidate_account_status.unblocked? %>
+      <%= govuk_link_to 'Block Account', support_interface_edit_candidate_account_status_path(@candidate) %>
+    <% else %>
+      <span class="govuk-!-margin-right-2 app-link--warning">
+        <%= SupportInterface::CandidateAccountStatus.human_attribute_name(@candidate_account_status.status) %>
+      </span>
+      <%= govuk_link_to 'Change', support_interface_edit_candidate_account_status_path(@candidate) %>
+    <% end %>
+  </div>
+<% end %>
```

Comment:
Not essential but might be nice to extract this block of markup into a component. (The feature flag check could sit in the `render?` method).

Reply:
Done! ♻️